### PR TITLE
stable 31.20200223.3.0 and testing 31.20200310.2.0 rollouts

### PR DESCRIPTION
**Pull request by jlebon**

### CODE REVIEW

**Comment by jlebon** [comment]
```
stable rollout of 31.20200223.3.0 scheduled for 2020-03-12 17:30:00 UTC for 4320m (72.0h)
testing rollout of 31.20200310.2.0 scheduled for 2020-03-12 17:30:00 UTC for 1440m (24.0h)
```

**Comment by jlebon** [comment]
Guess we could make the stable rollout shorter since it's going to leak into the weekend as it is. 48h?

File: updates/stable.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
         }
       }
     },
     {
-      "version": "31.20200210.3.0",
+      "version": "31.20200223.3.0",
       "metadata": {
         "rollout": {
-          "start_epoch": 1582655400,
+          "start_epoch": 1584034200,
```
Comment (dustymabe) [inline updates/stable.json]:
```
$ date -d @1584034200
Thu 12 Mar 2020 01:30:00 PM EDT
```

👍

File: updates/testing.json [hunk 2]
```diff
@@ -21,18 +21,18 @@
...
         }
       }
     },
     {
-      "version": "31.20200223.2.0",
+      "version": "31.20200310.2.0",
       "metadata": {
         "rollout": {
-          "start_epoch": 1582655400,
+          "start_epoch": 1584034200,
```
Comment (dustymabe) [inline updates/testing.json]:
```
$ date -d @1584034200
Thu 12 Mar 2020 01:30:00 PM EDT
```

👍

**Comment by dustymabe** [comment]
> Guess we could make the stable rollout shorter since it's going to leak into the weekend as it is. 48h?

yeah probably a good idea

**Comment by jlebon** [comment]
OK, updated for that. No other change, merging!

**Pull request merged**